sideboard/lib/__init__.py

```diff
@@ -16,7 +17,8 @@
            'ConfigurationError', 'parse_config',
            'is_listy', 'listify', 'serializer', 'cached_property', 'class_property', 'entry_point',
            'stopped', 'on_startup', 'on_shutdown', 'mainloop', 'ajax', 'renders_template', 'render_with_templates',
-           'restricted', 'all_restricted', 'register_authenticator'
```

Comment:
Funny this line was missing a comma, so I think the two items were concatenated and `register_authenticatorDaemonTask` was part of `__all__`.

On a stylistic note, in my other projects I've started defining `__all__` in my submodules and then doing `import *` make the submodule exports available in the parent module. For example inside `sideboard.lib._profiler` I'll have:
```python
__all__ = ['cleanup_profiler', 'profile', 'Profiler', 'ProfileAggregator']
```
and then in the parent module I'll have:
```python
from sideboard.lib._profiler import *
```
That way each individual module is responsible for what it is exporting, and the parent module can easily import everything for easy access elsewhere. You get the same functionality we have here, but without an unwieldy `__all__` list.